.cirrus.yml

```diff
@@ -95,6 +99,9 @@ task:
     chown root:postgres /
     chmod g+rwx /
 
+  install_meson_and_rebase_script: |
+    bash src/tools/ci/install_meson_and_rebase.sh ${MESON_REPO} ${MESON_BRANCH} linux
```

Comment:
Could we use ${CIRRUS_OS} instead of needing to write the os ourselves?

Reply:
Also, I think the script already would get $MESON_BRANCH etc as an environment variable, so we don't need to explicitly pass it. Normally explicit is better, but that way we don't need to deal with differences between shells etc?